rules/apps.libsonnet

```diff
@@ -63,20 +63,28 @@
           {
             record: 'namespace:kube_pod_container_resource_requests_memory_bytes:sum',
             expr: |||
-              sum by (namespace, label_name) (
-                  sum(kube_pod_container_resource_requests_memory_bytes{%(kubeStateMetricsSelector)s} * on (endpoint, instance, job, namespace, pod, service) group_left(phase) (kube_pod_status_phase{phase=~"Pending|Running"} == 1)) by (namespace, pod)
-                * on (namespace, pod)
-                  group_left(label_name) kube_pod_labels{%(kubeStateMetricsSelector)s}
+              sum by (namespace) (
+                  sum by (namespace, pod) (
+                      max by (namespace, pod, container) (
+                          kube_pod_container_resource_requests_memory_bytes{%(kubeStateMetricsSelector)s}
+                      ) * on(namespace, pod) group_left() max by (namespace, pod) (
```

Comment:
I’m curious if we even need the group_left here still or would a normal multiplication be sufficient?

Reply:
Correct (at least according to the tests :-). That was an left-over from a previous version of the patch.